common/jmeter/saml/SAML2_SSO_Redirect_Binding.jmx

```diff
@@ -207,7 +212,7 @@ import java.util.zip.DeflaterOutputStream;
 
 String acs = vars.get(&quot;assertionConsumerUrlPrefix&quot;) + &quot;_&quot; + vars.get(&quot;spId&quot;) + &quot;.&quot; + vars.get(&quot;assertionConsumerUrlSufix&quot;);
 String issuer = vars.get(&quot;issuerPrefix&quot;) + &quot;_&quot; + vars.get(&quot;spId&quot;);
-String xml = &quot;&lt;?xml version=\&quot;1.0\&quot; encoding=\&quot;UTF-8\&quot;?&gt;&lt;samlp:AuthnRequest xmlns:samlp=\&quot;urn:oasis:names:tc:SAML:2.0:protocol\&quot; AssertionConsumerServiceURL=\&quot;&quot; + acs + &quot;\&quot; Destination=\&quot;https://localhost:9443/samlsso\&quot; ForceAuthn=\&quot;false\&quot; ID=\&quot;0\&quot; IsPassive=\&quot;false\&quot; IssueInstant=\&quot;2017-08-31T09:34:51.710Z\&quot; ProtocolBinding=\&quot;urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\&quot; Version=\&quot;2.0\&quot;&gt;&lt;samlp:Issuer xmlns:samlp=\&quot;urn:oasis:names:tc:SAML:2.0:assertion\&quot;&gt;&quot; + 
+String xml = &quot;&lt;?xml version=\&quot;1.0\&quot; encoding=\&quot;UTF-8\&quot;?&gt;&lt;samlp:AuthnRequest xmlns:samlp=\&quot;urn:oasis:names:tc:SAML:2.0:protocol\&quot; AssertionConsumerServiceURL=\&quot;&quot; + acs + &quot;\&quot; Destination=\&quot;https://localhost:9443/samlsso\&quot; ForceAuthn=\&quot;false\&quot; ID=\&quot;0\&quot; IsPassive=\&quot;false\&quot; IssueInstant=\&quot;2017-08-31T09:34:51.710Z\&quot; ProtocolBinding=\&quot;urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\&quot; Version=\&quot;2.0\&quot;&gt;&lt;samlp:Issuer xmlns:samlp=\&quot;urn:oasis:names:tc:SAML:2.0:assertion\&quot;&gt;&quot; +
```

Comment:
can you explain what is this change ?

Reply:
Removed by https://github.com/wso2/performance-is/pull/128/commits/ef88a709e62a267d93e2cf768f66851ed0a8035e